packages/core/src/__tests__/document.test.ts

```diff
@@ -18,7 +18,7 @@ import ThreeIdResolver from '@ceramicnetwork/3id-did-resolver'
 
 jest.mock('../store/level-state-store')
 
-import InMemoryAnchorService from "../anchor/memory/in-memory-anchor-service"
+import InMemoryAnchorService, { IN_MEMORY_ANCHOR_SERVICE_CHAIN_ID } from "../anchor/memory/in-memory-anchor-service"
```

Comment:
Including `IN_MEMORY_ANCHOR_SERVICE_CHAIN_ID` but not using it in this file.

Reply:
👍